photon-core/src/main/java/org/photonvision/common/hardware/PiVersion.java

```diff
@@ -34,7 +34,7 @@ public enum PiVersion {
     private static final ShellExec shell = new ShellExec(true, false);
     private static final PiVersion currentPiVersion = calcPiVersion();
 
-    private PiVersion(String s) {
+    PiVersion(String s) {
```

Comment:
I want this to be private tho?

Reply:
I might be mistaken but enum constructors are default private so declaring them private is redundant.

Reply:
I don't remember that -- do you have a source?

Reply:
afaik enums cannot be constructed outside of its initial definition, so their constructor accessibility doesn't really matter

Reply:
You can't invoke the enum constructor beside within the enum itself, so it doesn't matter. The private modifier is redundant as it is already private.

Reply:
But it's not -wrong- really, is it?